packages/client/src/client/styleguide/sections/FontFamilies.tsx

```diff
@@ -42,153 +36,95 @@ const CHARACTERS = [
   `‘?’“!”(%)[#]{@}/&<-+÷×=>®©$€£¥¢:;,.*`,
 ].map((line) => line.split("").join(" "))
 
-const ROBOTO = {
-  fontFamily: "roboto",
-  fontFaces: [
-    { fontWeight: 100, advised: true, name: "Thin" },
-    { fontWeight: 200, advised: true, name: "Extra Light" },
-    { fontWeight: 300, advised: true, name: "Light" },
-    { fontWeight: 400, advised: true, name: "Regular" },
-    { fontWeight: 500, advised: true, name: "Medium" },
-    { fontWeight: 600, advised: true, name: "Semi Bold" },
-    { fontWeight: 700, advised: true, name: "Bold" },
-    { fontWeight: 800, advised: true, name: "Extra Bold" },
-    { fontWeight: 900, advised: true, name: "Black" },
-  ],
-}
-
 export const FontFamilies = () => (
   <>
     <SectionBlock>
-      <H2 pt={4}>Typography</H2>
-      <H3>Roboto</H3>
-
-      <Paragraph>
-        Roboto has a dual nature. It has a mechanical skeleton and the forms are
-        largely geometric. At the same time, the font features friendly and open
-        curves. While some grotesks distort their letterforms to force a rigid
-        rhythm, Roboto doesn’t compromise, allowing letters to be settled into
-        their natural width. This makes for a more natural reading rhythm more
-        commonly found in humanist and serif types.
-      </Paragraph>
-
-      <FontFamilySample {...ROBOTO} />
-
-      <H3>Font Sizes & Style Usage</H3>
-      <FontSizeGridHeader>
-        <H5>Category</H5>
-        <H5>Font Styles</H5>
-        <H5>Alignment</H5>
-        <H5>Size</H5>
-        <H5>Case</H5>
-      </FontSizeGridHeader>
-      {SIZES.map(
-        ([label, fontWeight, alignment, fontSizeText, caps, fontSize]) => (
-          <FontSizeGrid key={label}>
-            <Text
-              textColor={(theme) => theme.secondary[1]}
-              fontSize={fontSize}
-              lineHeight={1.5}
-            >
-              {label}
-            </Text>
-            <Text textColor={(theme) => theme.secondary[1]}>{fontWeight}</Text>
-            <Text textColor={(theme) => theme.secondary[1]}>{alignment}</Text>
-            <Text textColor={(theme) => theme.secondary[1]}>
-              {fontSizeText}
-            </Text>
-            <Text textColor={(theme) => theme.secondary[1]}>{caps}</Text>
-          </FontSizeGrid>
-        ),
-      )}
+      <H2>Typography</H2>
+      <H3>Work Sans</H3>
+
```

Comment:
as we have lost the blurb on Roboto, how about linking to the Google Font page for Work Sans?